.circleci/integration/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+gem install pkg/fluent-plugin-splunk-hec-1.0.1.gem
```

Comment:
This should be the latest one built from the code

Reply:
It is. It gets build during build step.

Reply:
agreed, as we start building more frequently. Need an intelligent way to track latest release/artifact version and use it.

Reply:
dont hardcode version number